src/__test__/pages/experiments/[experimentId]/data-processing/configure-embedding/components/CalculationConfig.test.jsx

```diff
@@ -54,10 +57,151 @@ describe('CalculationConfig', () => {
     fetchMock.mockResolvedValue(response);
   });
 
-  it('renders correctly when nothing is loaded', () => {
+  // it('renders correctly when nothing is loaded', () => {
+  //   const store = mockStore({
+  //     embeddings: {},
+  //     experimentSettings: {
+  //       ...initialExperimentState,
+  //     },
+  //   });
+
+  //   const component = mount(
+  //     <Provider store={store}>
+  //       <CalculationConfig
+  //         experimentId='1234'
+  //         width={50}
+  //         height={50}
+  //       />
+  //     </Provider>,
+  //   );
+
+  //   const spin = component.find(Spin);
+
+  //   // There should be a spinner for loading state.
+  //   expect(spin.length).toEqual(1);
+  // });
+
+  // it('renders correctly when the data is in the store', () => {
+  //   const store = mockStore(storeState);
+
+  //   const component = mount(
+  //     <Provider store={store}>
+  //       <CalculationConfig
+  //         experimentId='1234'
+  //         width={50}
+  //         height={50}
+  //       />
+  //     </Provider>,
+  //   );
+
+  //   // There should no spinner anymore.
+  //   const spin = component.find(Spin);
+  //   expect(spin.length).toEqual(0);
+
+  //   // There should be a form loaded.
+  //   const form = component.find(Form);
+  //   expect(form.length).toBeGreaterThan(0);
+  // });
+
+  // it('changing an embedding setting should trigger an alert', () => {
+  //   const store = mockStore(storeState);
+
+  //   const component = mount(
+  //     <Provider store={store}>
+  //       <CalculationConfig
+  //         experimentId='1234'
+  //         width={50}
+  //         height={50}
+  //       />
+  //     </Provider>,
+  //   );
+
+  //   // There should no spinner anymore.
+  //   const spin = component.find(Spin);
+  //   expect(spin.length).toEqual(0);
+
+  //   // There should be a form loaded.
+  //   const form = component.find(Form);
+  //   expect(form.length).toBeGreaterThan(0);
+
+  //   // There should be no alert loaded.
+  //   let alert = component.find(Alert);
```

Comment:
why is this commented out?

Reply:
Oh ya I forgot to remove the commenting out while I was testing. I'll fix it,